README.md

```diff
@@ -35,6 +35,8 @@ This API can be used to convert British National Grid eastings and northings to
 To complete the challenge you will need to:
 
 - Write a script called `enrich_data.py` to pull data from the API and enrich the data in the database with the latitudes and longitudes of the captures.
+    - it may take several minutes to receive the data from the API
```

Comment:
Keen to omit this as I'd like to see how people work through this. Really great responses will realise this and may try parallelise. We can talk people through it on the day if we need to

Reply:
Makes sense, my intent was to confirm that the API does indeed behave that way. But yes, can be vocal